rpc/cc/arduino/cli/monitor/v1/monitor.proto

```diff
@@ -21,7 +21,10 @@ option go_package = "github.com/arduino/arduino-cli/rpc/cc/arduino/cli/monitor/v
 
 import "google/protobuf/struct.proto";
 
-// MonitorService provides services for boards monitor
+// MonitorService provides services for boards monitor.
+// DEPRECATION WARNING: MonitorService is deprecated and will be removed in a
+// future release. Use CommandService.Monitor and
+// CommandService.EnumerateMonitorPortSettings instead.
```

Comment:
```suggestion
// future release. Use ArduinoCoreService.Monitor and
// ArduinoCoreService.EnumerateMonitorPortSettings instead.
```
Isn't this the service name?
https://arduino.github.io/arduino-cli/dev/rpc/commands/#cc.arduino.cli.commands.v1.ArduinoCoreService

Reply:
🤦🏼 yes correct